### DISCUSSION TITLE
Recolhimento de imposto retroativo

### DISCUSSION THREAD

**Discussion by bunnmv** [DyegoCosta/trabalhando-remoto#203] [General]
### Agradecimentos 
Fala galera! Queria começar parabenizando vocês por essa iniciativa de juntar informações tornar o processo colaborativo, uma pena eu ter achado isso tudo recentemente! Muita coisa consegui achar aqui nos fóruns mas estou com essa dúvida em aberto que gostaria de compartilhar com vocês e quem sabe alguém que já passou pelo mesmo, ou tenha mais experiência, possa me ajudar com essa.

### O problema

Vamos lá, minha dúvida é a seguinte. Desde Fevereiro venho prestando serviços no EUA, alguns pela UpWork e outros com contato direto. Para receber os valores criei uma conta TransferWise, PF, onde recebi todas as quantias. Desde o inicio minha intenção era deixar os proventos lá, na Borderless Account, assumi que não teria problema algum e apenas iria declarar a conta no meu IRPF no ano que vem, normalmente.. 

Nesse mês precisei resgatar uma parcela e resolvi me atualizar de como fazer, pesquisei muito e tudo é muito confuso,  consultei alguns contadores e também ficam meio perdidos com isso, mas aqui no repositório encontrei as melhores informações. 

Na discussão feita em [Emissão de Nota Fiscal para o exterior #102](https://github.com/DyegoCosta/trabalhando-remoto/issues/102 ), entendi que alguns declaram o valor depois que é convertido em reais, assim que quantia é trazida para uma conta no Brasil. Outros emitem o invoice e já convertem a quantia em reias para fazer a declaração, independente se vão trazer o dinheiro para o Brasil ou não. Acredito que me encaixo na segundo situação, e deveria declarar tudo o que entra na conta estrangeira.

### Situação atual

Sou MEI, mas meus serviços no exterior são de Desenvolvimento Web, sei que não é o certo e já é de minha intenção adequar a PJ para o ano que vem e começar tudo certinho. No entanto, **como eu não declarei nada na MEI, não emiti nenhuma NF**, fico na dúvida de como proceder neste momento. Tenho quatro ideias principais aqui. Poderiam me ajudar a definir o caminho mais correto?

### 1 - Carnê Leão Retroativo e seguir com ele

Aqui eu iria acessar o sistema do Carnê Leão e gerar as guias de pagamento desde Fevereiro até aqui e ajustar os pagamentos com base nas multas calculadas pelo [Sicalc](http://receita.economia.gov.br/orientacao/tributaria/pagamentos-e-parcelamentos/darf-calculo-e-impressao-programa-sicalc-1/darf-calculo-e-impressao-programa-sicalc), pagar tudo e pedir desculpas ao Fisco. Em alguns meses a quantia foi pouca e nem chegou aos R$ 1.903,98 (valor mínimo obrigatório para emissão do Carnê Leão), outros passaram da alíquota de 27.5 %.

Para este mês e os próximos, seguir no carnê leão, e com a vantagem de não pagar multas.

### 2 - Carnê Leão Retroativo e NF para este mês e os próximos

Mesmo procedimento do anterior para os meses de Fevereiro à Setembro. E começar a emitir NF daqui para frente como MEI, até que eu crie a minha ME no próximo ano fiscal e mantenha este procedimento.

### 3 - Deixar tudo para o IPRF

Já vi aqui no repositório que está não é uma opção muito boa, mas também não sei se o valor final seria superior ao valor pago pelas multas no carnê leão em atraso. Acredito que pagar de forma voluntária, na opção 1, seja a melhor. Do que ser intimado. 

### 4 - Não emitir NF e deixar isso na MEI

Simplesmente não emitir notas e declarar as quantias todas no DASN SIMEI. Como MEI teria que emitir nota aos clientes PJ, que foi a maioria dos casos já que apenas dois foram clientes PF. Então, acredito que isso possa criar problemas e gerar multas de não ter emitido notas a quem era devido (clientes PJ).

### O que penso

Bom, minha opinião é que o mais correto seria tratar tudo como PF e efetuar os procedimentos descritos na situação 1. Mas como estou cheio de dúvidas gostaria da opinião de vocês sobre como proceder entre as opções mencionadas ou alguma nova sugestão.

**Comment by samiraguiar** [comment]
@bunnmv eu entendo que a opção I seja a melhor também. Legalmente falando, MEI não pode exercer atividades de desenvolvimento de software. Tem [um projeto de lei](https://www25.senado.leg.br/web/atividade/materias/-/materia/133127) em tramitação pra permitir isso, mas ainda não foi aprovado. Então não seria correto você jogar pra sua PJ, ainda mais pelo fato de você receber em conta PF. Tem também a questão dos contratos com dos seus trabalhos: se o contrato foi feito com o seu CPF, não seria possível jogar de forma retroativa pra MEI.
Ou seja, se fosse pra deixar redondo, eu regularizaria a situação na PF e seguiria como a PF msmo enquanto valesse a pena (comparado com uma empresa do Simples, por exemplo).

**Comment by bunnmv** [comment]
Certo, faz todo sentido!

Obrigado @samiraguiar !

**Comment by elMor3no** [comment]
Muito obrigado... 
Também tinha uma duvida muito similar
